src/CLRunner.ts

```diff
@@ -335,6 +335,70 @@ export class CLRunner {
         return startResponse
     }
 
+    private async StartSession(appId: string, createParams: CLM.SessionCreateParams): Promise<CLM.Session> {
+
+        // Don't save logs on server if custom storage was provided
+        if (CLState.logStorage) {
+            createParams.saveToLog = false
+        }
+
+        const session = await this.clClient.StartSession(appId, createParams as CLM.SessionCreateParams)
+
+        // If using customer storage add to log storage
+        if (CLState.logStorage) {
+            // For self-hosted log storage logDialogId is hash of sessionId
+            session.logDialogId = CLM.hashText(session.logDialogId)
+            const logDialog: CLM.LogDialog = {
+                logDialogId: session.logDialogId,
+                packageId: session.packageId,
+                rounds: [],
+                initialFilledEntities: [],
+                targetTrainDialogIds: [],
+                createdDateTime: session.createdDatetime,
+                dialogBeginDatetime: new Date().getTime().toString(),
```

Comment:
Is this format of date required?  Since it is creating JSON object we should change it to use the standard. The `toJSON()` does the ISO 8601 form `2019-11-06T00:41:58.669Z`  instead of `1573000963058`

Reply:
done